[qtbase] Build failue

Package: qtbase[core,doubleconversion,freetype,gui,harfbuzz,thread]:arm64-osx -> 6.5.0

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: a5d91f7d2 2023-05-01 (10 hours ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`

**Failure logs**

```
-- Using cached qtbase-everywhere-src-6.5.0.tar.xz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/qtbase-everywhere-src-6.5.0.tar.xz
-- Applying patch allow_outside_prefix.patch
-- Applying patch config_install.patch
-- Applying patch fix_cmake_build.patch
-- Applying patch harfbuzz.patch
-- Applying patch fix_egl.patch
-- Applying patch fix_egl_2.patch
-- Applying patch installed_dir.patch
-- Applying patch GLIB2-static.patch
-- Applying patch clang-cl_source_location.patch
-- Applying patch clang-cl_QGADGET_fix.diff
-- Applying patch dont_force_cmakecache_latest.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean
-- Found external ninja('1.11.1').
-- Configuring arm64-osx
-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/qtbase/cmake/qt_install_submodule.cmake:338 (vcpkg_cmake_install)
  ports/qtbase/portfile.cmake:284 (qt_install_submodule)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/install-arm64-osx-dbg-out.log</summary>

```
[1/821] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/xml && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake "-DIN_FILES=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/xml/qt_lib_xml_private.pri;/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/xml/Debug/qt_lib_xml_private.cmake" -DOUT_FILE=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/../share/Qt6/mkspecs/modules/qt_lib_xml_private.pri "-DLIBRARY_PREFIXES=lib;lib" "-DLIBRARY_SUFFIXES=.dylib;.tbd;.so;.a" -DLINK_LIBRARY_FLAG=-l -DCONFIGS=Debug -DIMPLICIT_LINK_DIRECTORIES=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib -P /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/cmake/QtGenerateLibPri.cmake
[2/821] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake "-DIN_FILES=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/mkspecs/preliminary/qmodule.pri;/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/mkspecs/preliminary/Debug/qmodule.cmake" -DOUT_FILE=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/../share/Qt6/mkspecs/qmodule.pri "-DLIBRARY_PREFIXES=lib;lib" "-DLIBRARY_SUFFIXES=.dylib;.tbd;.so;.a" -DLINK_LIBRARY_FLAG=-l -DCONFIGS=Debug -DIMPLICIT_LINK_DIRECTORIES=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib -P /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/cmake/QtGenerateLibPri.cmake
[3/821] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake "-DIN_FILES=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/qt_lib_gui_private.pri;/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Debug/qt_lib_gui_private.cmake" -DOUT_FILE=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/../share/Qt6/mkspecs/modules/qt_lib_gui_private.pri "-DLIBRARY_PREFIXES=lib;lib" "-DLIBRARY_SUFFIXES=.dylib;.tbd;.so;.a" -DLINK_LIBRARY_FLAG=-l -DCONFIGS=Debug -DIMPLICIT_LINK_DIRECTORIES=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib -P /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/cmake/QtGenerateLibPri.cmake
[4/821] cd /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake "-DIN_FILES=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib/qt_lib_core_private.pri;/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib/Debug/qt_lib_core_private.cmake" -DOUT_FILE=/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/../share/Qt6/mkspecs/modules/qt_lib_core_private.pri "-DLIBRARY_PREFIXES=lib;lib" "-DLIBRARY_SUFFIXES=.dylib;.tbd;.so;.a" -DLINK_LIBRARY_FLAG=-l -DCONFIGS=Debug -DIMPLICIT_LINK_DIRECTORIES=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/lib -P /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/cmake/QtGenerateLibPri.cmake
...
Skipped 764 lines
...

[685/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/painting/qrhibackingstore.cpp.o -MF src/gui/CMakeFiles/Gui.dir/painting/qrhibackingstore.cpp.o.d -o src/gui/CMakeFiles/Gui.dir/painting/qrhibackingstore.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/painting/qrhibackingstore.cpp
[686/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/painting/qrasterbackingstore.cpp.o -MF src/gui/CMakeFiles/Gui.dir/painting/qrasterbackingstore.cpp.o.d -o src/gui/CMakeFiles/Gui.dir/painting/qrasterbackingstore.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/painting/qrasterbackingstore.cpp
[687/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/platform/macos/qcocoanativeinterface.mm.o -MF src/gui/CMakeFiles/Gui.dir/platform/macos/qcocoanativeinterface.mm.o.d -o src/gui/CMakeFiles/Gui.dir/platform/macos/qcocoanativeinterface.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/platform/macos/qcocoanativeinterface.mm
FAILED: src/gui/CMakeFiles/Gui.dir/platform/macos/qcocoanativeinterface.mm.o 
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/platform/macos/qcocoanativeinterface.mm.o -MF src/gui/CMakeFiles/Gui.dir/platform/macos/qcocoanativeinterface.mm.o.d -o src/gui/CMakeFiles/Gui.dir/platform/macos/qcocoanativeinterface.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/platform/macos/qcocoanativeinterface.mm
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/platform/macos/qcocoanativeinterface.mm:4:10: fatal error: 'QtGui/private/qopenglcontext_p.h' file not found
#include <QtGui/private/qopenglcontext_p.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
[688/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/animation/qguivariantanimation.cpp.o -MF src/gui/CMakeFiles/Gui.dir/animation/qguivariantanimation.cpp.o.d -o src/gui/CMakeFiles/Gui.dir/animation/qguivariantanimation.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/animation/qguivariantanimation.cpp
[689/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/platform/darwin/qmacmimeregistry.mm.o -MF src/gui/CMakeFiles/Gui.dir/platform/darwin/qmacmimeregistry.mm.o.d -o src/gui/CMakeFiles/Gui.dir/platform/darwin/qmacmimeregistry.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/platform/darwin/qmacmimeregistry.mm
[690/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/image/qimage_darwin.mm.o -MF src/gui/CMakeFiles/Gui.dir/image/qimage_darwin.mm.o.d -o src/gui/CMakeFiles/Gui.dir/image/qimage_darwin.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/image/qimage_darwin.mm
[691/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/itemmodels/qstandarditemmodel.cpp.o -MF src/gui/CMakeFiles/Gui.dir/itemmodels/qstandarditemmodel.cpp.o.d -o src/gui/CMakeFiles/Gui.dir/itemmodels/qstandarditemmodel.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/itemmodels/qstandarditemmodel.cpp
[692/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/painting/qcoregraphics.mm.o -MF src/gui/CMakeFiles/Gui.dir/painting/qcoregraphics.mm.o.d -o src/gui/CMakeFiles/Gui.dir/painting/qcoregraphics.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/painting/qcoregraphics.mm
[693/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/platform/darwin/qutimimeconverter.mm.o -MF src/gui/CMakeFiles/Gui.dir/platform/darwin/qutimimeconverter.mm.o.d -o src/gui/CMakeFiles/Gui.dir/platform/darwin/qutimimeconverter.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/platform/darwin/qutimimeconverter.mm
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/platform/darwin/qutimimeconverter.mm:766:73: warning: 'kUTTypeTIFF' is deprecated: first deprecated in macOS 12.0 - Use UTTypeTIFF instead. [-Wdeprecated-declarations]
                                                                        kUTTypeTIFF, 1, 0);
                                                                        ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/System/Library/Frameworks/CoreServices.framework/Frameworks/LaunchServices.framework/Headers/UTCoreTypes.h:726:26: note: 'kUTTypeTIFF' has been explicitly marked deprecated here
extern const CFStringRef kUTTypeTIFF                                 API_DEPRECATED("Use UTTypeTIFF instead.", ios(3.0, 15.0), macos(10.4, 12.0), tvos(9.0, 15.0), watchos(1.0, 8.0));
                         ^
1 warning generated.
[694/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/platform/darwin/qapplekeymapper.mm.o -MF src/gui/CMakeFiles/Gui.dir/platform/darwin/qapplekeymapper.mm.o.d -o src/gui/CMakeFiles/Gui.dir/platform/darwin/qapplekeymapper.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/platform/darwin/qapplekeymapper.mm
[695/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -fPIC -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/accessible/qaccessible.cpp.o -MF src/gui/CMakeFiles/Gui.dir/accessible/qaccessible.cpp.o.d -o src/gui/CMakeFiles/Gui.dir/accessible/qaccessible.cpp.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/accessible/qaccessible.cpp
[696/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/text/coretext/qfontengine_coretext.mm.o -MF src/gui/CMakeFiles/Gui.dir/text/coretext/qfontengine_coretext.mm.o.d -o src/gui/CMakeFiles/Gui.dir/text/coretext/qfontengine_coretext.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/text/coretext/qfontengine_coretext.mm
[697/821] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ -DGL_SILENCE_DEPRECATION -DMD4C_USE_UTF8 -DQT_ASCII_CAST_WARNINGS -DQT_BUILDING_QT -DQT_BUILD_GUI_LIB -DQT_CORE_LIB -DQT_DEPRECATED_WARNINGS -DQT_DISABLE_DEPRECATED_UP_TO=0x050000 -DQT_EXPLICIT_QFILE_CONSTRUCTION_FROM_PATH -DQT_LEAN_HEADERS=1 -DQT_MOC_COMPAT -DQT_NO_AS_CONST=1 -DQT_NO_CAST_TO_ASCII -DQT_NO_EXCEPTIONS -DQT_NO_FOREACH -DQT_NO_JAVA_STYLE_ITERATORS -DQT_NO_NARROWING_CONVERSIONS_IN_CONNECT -DQT_NO_USING_NAMESPACE -DQT_QPA_DEFAULT_PLATFORM_NAME=\"cocoa\" -DQT_USE_QSTRINGBUILDER -DQT_WARN_DEPRECATED_UP_TO=0x070000 -D_LARGEFILE64_SOURCE -D_LARGEFILE_SOURCE -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/Gui_autogen/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/VulkanMemoryAllocator -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtGui/6.5.0/QtGui -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/../3rdparty/md4c -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/mkspecs/macx-clang -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/corelib -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0 -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/include/QtCore/6.5.0/QtCore -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/harfbuzz -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include -x objective-c++ -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk -fPIC -fvisibility=hidden -fvisibility-inlines-hidden -Wall -Wextra -fno-exceptions -ffunction-sections -fdata-sections -fapplication-extension -std=c++17 -Winvalid-pch -Xarch_arm64 -include/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/arm64-osx-dbg/src/gui/CMakeFiles/Gui.dir/cmake_pch_arm64.objcxx.hxx -MD -MT src/gui/CMakeFiles/Gui.dir/text/coretext/qcoretextfontdatabase.mm.o -MF src/gui/CMakeFiles/Gui.dir/text/coretext/qcoretextfontdatabase.mm.o.d -o src/gui/CMakeFiles/Gui.dir/text/coretext/qcoretextfontdatabase.mm.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtbase/src/here-src-6-f44b1e9782.clean/src/gui/text/coretext/qcoretextfontdatabase.mm
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "qtbase",
      "default-features": false,
      "features": [
        "harfbuzz"
      ]
    }
  ],
  "$overrides": [
    {
      "name": "capstone",
      "version": "4.0.1-120373dc"
    }
  ],
  "$builtin-baseline": "338f99eed499b6df1628829e40695b396a70a1f7"
}

```
</details>

--- a/ports/qtbase/vcpkg.json
+++ b/ports/qtbase/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "qtbase",
   "version": "6.5.0",
+  "port-version": 1,
   "description": "Qt Application Framework Base Module. Includes Core, GUI, Widgets, Networking, SQL, Concurrent and other essential qt components.",
   "homepage": "https://www.qt.io/",
   "license": null,
@@ -207,6 +208,14 @@
           "features": [
             "freetype"
           ]
+        },
+        {
+          "name": "qtbase",
+          "default-features": false,
+          "features": [
+            "opengl"
+          ],
+          "platform": "osx"
         },
         {
           "name": "qtbase",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6638,7 +6638,7 @@
     },
     "qtbase": {
       "baseline": "6.5.0",
-      "port-version": 0
+      "port-version": 1
     },
     "qtcharts": {
       "baseline": "6.5.0",

--- a/versions/q-/qtbase.json
+++ b/versions/q-/qtbase.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "a66d81c3d9432499f24c76df346a4daf5725f90b",
+      "version": "6.5.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "501903e9d1d55ca5dbeee2ff61c8c846e77fe7b3",
       "version": "6.5.0",
       "port-version": 0

